packages/design/src/Link/Link.jsx

```diff
@@ -29,19 +29,17 @@ Link.defaultProps = {
 
 Link.displayName = 'Link';
 
-const StyledButtonLink = styled.a`
+const StyledButtonLink = styled.a.attrs({
+  rel: 'noreferrer',
+})`
   color: ${({ theme }) => theme.colors.link};
   font-weight: normal;
   background: none;
   text-decoration: underline;
   text-transform: none;
 
-  &:hover,
-  &:focus {
-    background: ${({ theme }) => theme.colors.primary.light};
-  }
```

Comment:
The highlighting affect for hover/focus had no affect b/c the color is the same as the content background color so I removed it